src/pages/theming/dark-mode.md

```diff
@@ -116,6 +116,23 @@ function checkToggle(shouldCheck) {
 <!-- Codepen https://codepen.io/ionic/pen/zYOpQLj -->
 <docs-codepen preview="false" user="ionic" slug="zYOpQLj" height="600px" default-tab="js,result"></docs-codepen>
 
+## Adjusting System UI Components
+
+When developing a dark theme, you may notice that certain system UI components are not adjusting to dark mode properly. To fix this you will need to specify the `color-scheme`. At the time of writing, this is supported in Safari and will be implemented in Chrome soon.
+
+While you may be using Ionic components instead of just the native components, `color-scheme` can also affect aspects of your application such as the scrollbar. In order to use `color-scheme` you will need to add the following HTML to the `head` of your application:
+
+```html
+<meta name="color-scheme" content="light dark" />
+```
+
+This allows the page to indicate which color scheme it is comfortable being rendered with. Alternatively, you can add the following CSS to do this on a per-element basis:
+
+```css
+color-scheme: light dark;
+```
+
+<docs-codepen preview="false" user="ionic" slug="dyorvLd" height="300px" default-tab="result"></docs-codepen>
```

Comment:
I would love some feedback on this codepen re: how I can spruce it up and not make it look so bland

Reply:
Does this need a Codepen? What if we made it an image that shows scrollbars in each mode or something?